Script/AtomicEditor/hostExtensions/languageExtensions/TypscriptLanguageExtension.ts

```diff
@@ -337,6 +341,25 @@ export default class TypescriptLanguageExtension implements Editor.HostExtension
         WebView.WebBrowserHost.setGlobalStringProperty("TypeScriptLanguageExtension", "tsConfig", JSON.stringify(tsConfig));
     }
 
+    generateExternalProject ()
+    {
+      var projectDir : string = ToolCore.toolSystem.project.projectPath;
+
+      // Create TypeScript folder in project root
+      var projectDirTypeScript : string = Atomic.addTrailingSlash(projectDir + "TypeScript");
+      Atomic.getFileSystem().createDir(projectDirTypeScript);
+
+      // Copy the Atomic.d.ts definition file to the TypeScript folder
+      var toolDataDir : string = ToolCore.toolEnvironment.toolDataDir;
+      var typescriptSupportDir : string = Atomic.addTrailingSlash(toolDataDir + "TypeScriptSupport");
+      Atomic.getFileSystem().copy(typescriptSupportDir + "Atomic.d.ts", projectDirTypeScript + "Atomic.d.ts");
+
+      // Generate a tsconfig.json file
+      var config = new Atomic.File(projectDir + "tsconfig.json", Atomic.FILE_WRITE);
+      config.writeString(JSON.stringify(defaultCompilerOptions));
+      config.close();
+
```

Comment:
I took a look at the generated ```json``` file and the ```compiler options``` need to be under their own node.  So, something like:
```
let tsconfig = {
  compilerOptions: defaultCompilerOptions
};
config.writeString(JSON.stringify(tsconfig));
```
should work